packages/client/src/orders/getGuestOrder.ts

```diff
@@ -0,0 +1,35 @@
+import client, { adaptError } from '../helpers/client';
+import isString from 'lodash/isString';
+import join from 'proper-url-join';
+import type { GetGuestOrder, GetGuestOrdersQuery } from './types';
+
+/**
+ * Method responsible for fetching the details of an guest user order.
+ *
+ * @function getGuestOrder
+ * @memberof module:orders/client
```

Comment:
I think this syntax is no longer correct for tsdoc. Refer to https://github.com/Farfetch/blackout/pull/328/ for an example of this